MonacoEditorComponent/CodeEditor.cs

```diff
@@ -140,7 +140,10 @@ private void CodeEditor_Unloaded(object sender, RoutedEventArgs e)
             _parentAccessor?.Dispose();
             _parentAccessor = null;
             Options.PropertyChanged -= Options_PropertyChanged;
-            _themeListener.ThemeChanged -= _themeListener_ThemeChanged;
+
+            if(_themeListener != null)
```

Comment:
I believe you could use this syntax to make it easier:

```csharp
_themeListener?.ThemeChanged -= _themeListener_ThemeChanged;
```

Reply:
Yeah

Reply:
Nope, not supported ☹. Would've been nice though.